### PR TITLE
Scroll to top button (#665)

### PR DESCRIPTION
**Pull request by varunwho**
### Scroll to top button

https://user-images.githubusercontent.com/53107832/122255536-d8828180-ceeb-11eb-8b3b-5608f0b37c58.mp4

### CODE REVIEW

File: assets/js/dynamicTitle.js [hunk 1]
```diff
@@ -1,5 +1,3 @@
-
```
Comment (smaranjitghose) [inline assets/js/dynamicTitle.js:1]: Avoid these changes for now

File: assets/js/app.js [hunk 2]
```diff
@@ -213,13 +213,18 @@ let footer = $(`
 </footer>
 `);
 
-
```
Comment (smaranjitghose) [inline assets/js/app.js:216]: Add appropriate comments

**Review by smaranjitghose — CHANGES_REQUESTED** [review]

**Comment by smaranjitghose** [comment]
Reduce the size of the button by 25% of it's present size. The color of the button should be different while on the footer and on the remainder of the page

**Comment by varunwho** [comment]
> Reduce the size of the button by 25% of it's present size. The color of the button should be different while on the footer and on the remainder of the page

Yup I will do it

**Review by anushbhatia — CHANGES_REQUESTED** [review]
On selection of the button avoid that black box around it if possible.

**Comment by varunwho** [comment]
#665 
### **I've done these changes**
- Reduced the size
- Added the microanimation which was suggested
- Removed the outline which appeared after the click
- The color of the button changes when it hits the footer

**Comment by smaranjitghose** [comment]
@varunwho This looks perfect. Just incorporate the recent changes to the project to remove the merge conflict.

**Review by smaranjitghose — APPROVED** [review]

**Review by anushbhatia — CHANGES_REQUESTED** [review]
When we move to the top of the page it should vanish automatically.

**Comment by varunwho** [comment]
Ignore the latest commit here, I've made a new PR

**Pull request closed without merging**